tests/unit/test1_unit_tests/src/main.c

```diff
@@ -67,30 +67,32 @@ void test_uint64(void)
 	uint64_t uint64;
 	uint32_t uint32;
 
-	ZCBOR_STATE_E(state_e, 0, payload, sizeof(payload), 0);
-	ZCBOR_STATE_D(state_d, 0, payload, sizeof(payload), 10);
-
-	zassert_true(zcbor_uint64_put(state_e, 5), NULL);
-	zassert_false(zcbor_uint64_expect(state_d, 4), NULL);
-	zassert_false(zcbor_uint64_expect(state_d, 6), NULL);
-	zassert_false(zcbor_uint64_expect(state_d, -5), NULL);
-	zassert_false(zcbor_uint64_expect(state_d, -6), NULL);
-	zassert_true(zcbor_uint64_expect(state_d, 5), NULL);
-
-	zassert_true(zcbor_uint32_put(state_e, 5), NULL);
-	zassert_true(zcbor_uint64_expect(state_d, 5), NULL);
-
-	zassert_true(zcbor_uint64_put(state_e, 5), NULL);
-	zassert_true(zcbor_uint32_expect(state_d, 5), NULL);
-
-	zassert_true(zcbor_uint64_put(state_e, UINT64_MAX), NULL);
-	zassert_false(zcbor_uint32_decode(state_d, &uint32), NULL);
-	zassert_true(zcbor_uint64_decode(state_d, &uint64), NULL);
+	zcbor_state_t state_e;
```

Comment:
Nothing wrong with this line. Is `ZCBOR_STATE_E/D` still used?

Reply:
It could be, but it's nice to try the different ways of using the API.